kratos/python_scripts/from_json_check_result_process.py

```diff
@@ -24,6 +24,47 @@ class FromJsonCheckResultProcess(KratosMultiphysics.Process, KratosUnittest.Test
     settings -- Kratos parameters containing solver settings.
     """
 
+    def __init__(self, model, params):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        model -- the model contaning the model_parts
+        settings -- Kratos parameters containing solver settings.
+        """
+        KratosMultiphysics.Process.__init__(self)
+        self.process = KratosMultiphysics.FromJSONCheckResultProcess(model, params)
```

Comment:
this should be returned in the `Factory` of this class, then there is no more python-instance left (where e.g. once could forget to call some methods)

Reply:
It is needed in order to call the unittest interface

Reply:
you are right, sorry for the noise